mydumper/mydumper.go

```diff
@@ -261,12 +262,18 @@ func (m *Mydumper) constructArgs() []string {
 	extraArgs := strings.Fields(cfg.ExtraArgs)
 	if len(extraArgs) > 0 {
 		ret = append(ret, ParseArgLikeBash(extraArgs)...)
+	} else {
```

Comment:
if extraArgs is not empty but like "--skip-tz-utc", we should also generate `--tables-list` for it